### PR TITLE
Add retry on test_task_creation

### PR DESCRIPTION
**Pull request by nikhil-thomas**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+#
+set -e
+
+function check-service-endpoints() {
+  service=${1}
+  namespace=${2}
+  echo "-----------------------"
+  echo "checking ${namespace}/${service} service endpoints"
+  count=0
+  while [[ -z $(kubectl get endpoints ${service} -n ${namespace} -o jsonpath='{.subsets}') ]]; do
```
Comment (chmouel) [inline openshift/e2e-tests-openshift.sh:13]: i think we could use kubectl wait for this, isnt it ?

**Comment by piyush-garg** [comment]
@nikhil-thomas is it still required ?

**Comment by nikhil-thomas** [comment]
obsolete

**Pull request closed without merging**